src/Features/Core/Portable/FeaturesResources.resx

```diff
@@ -885,4 +885,19 @@ Do you want to continue?</value>
   <data name="Make_method_synchronous" xml:space="preserve">
     <value>Make method synchronous.</value>
   </data>
+  <data name="from_0" xml:space="preserve">
+    <value>from {0}</value>
+  </data>
+  <data name="Find_and_install_latest_version" xml:space="preserve">
+    <value>Find and install latest version</value>
+  </data>
+  <data name="Use_local_version_0" xml:space="preserve">
+    <value>Use local version '{0}'</value>
+  </data>
+  <data name="Use_locally_installed_0_version_1" xml:space="preserve">
+    <value>Use locally installed '{0}' version '{1}'</value>
+  </data>
+  <data name="Find_and_install_latest_version_of_0" xml:space="preserve">
```

Comment:
"from NuGet.org"? Or are we trying to minimize width? 

Also, "Download and install latest version of '{0}'" would make it clear this isn't a local operation.